### ISSUE TITLE
[list] The 'list' command is a bit unhelpful outside a Drupal root

### ISSUE THREAD

**Issue by pjcdawkins** [hechoendrupal/DrupalAppConsole#463]
I get:

```
$ php console.phar list
Drupal Console version 0.7.2

Usage:
 command [options] [arguments]

Options:
 --help (-h)           Display this help message
 --quiet (-q)          Do not output any message
 --verbose (-v|vv|vvv) Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
 --version (-V)        Display this application version
 --ansi                Force ANSI output
 --no-ansi             Disable ANSI output
 --no-interaction (-n) Do not ask any interactive question
 --drupal (-d)         Path to Drupal root.
 --shell (-s)          Launch the shell.
 --env (-e)            The Environment name. (default: "prod")
 --no-debug            Switches off debug mode.

Available commands:
 help   Displays help for a command
 list   Lists commands
```

I feel I should probably get a list of available commands.

**Comment by pjcdawkins** [comment]
Updated title, as it appears that actually the commands just aren't available at all: I can only run `help` or `list`

**Comment by pjcdawkins** [comment]
Ah OK, I checked out 0.7.1, and that one has an exception saying "In order to list all of the available commands, you should run this inside a drupal root directory." Makes sense.. but that exception is no longer shown in 0.7.2. Can that bit of information be displayed in the "list" command when not inside a Drupal root?

**Comment by jmolivas** [comment]
Thanks for reporting @pjcdawkins, we will rollback the code displaying message `In order to list all of the available commands, you should run this inside a drupal root directory`

**Comment by jmolivas** [comment]
I am working one this o/

**Comment by jmolivas** [comment]
Fixed with PR #482 
Project shown available commands & message
```
 In order to list all of the available commands you should install drupal first.
```
![drupal-no-settings-file](https://cloud.githubusercontent.com/assets/366275/6451252/275d5fce-c0e2-11e4-83f9-2c385297ce49.png)
